packages/terra-clinical-result/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-clinical-result",
+  "main": "lib/ClincalResult.js",
+  "version": "0.0.0",
+  "description": "The Terra Clinical Result package is a collection of standardized views for presenting clinical results documented to a Patient's Medical Chart, such as Vital Signs, Laboratory Results, and Discretely Documented Values",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "terra-clinical-result",
+    "ClinicalResult",
+    "Result",
+    "Observation",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "^2.8.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-doc-template": "^2.12.0",
+    "terra-enzyme-intl": "^3.2.0",
```

Comment:
This should probably be a devDependency

Reply:
removed here: https://github.com/cerner/terra-clinical/pull/574/commits/4015e0f2acf433faa34429946f05e3db8a2ebebc#diff-f98243208c330dd1438f9a62456b5a8f